cfme/tests/test_ui_problems.py

```diff
@@ -12,7 +12,8 @@
 TOLERANCE = 20
 # TODO: Replace nav branch strings with navmazing object destinations when supported
 LOCATIONS = [
-    "control_explorer", "automate_explorer", "automate_customization", "my_services",
+    "control_explorer", "automate_explorer", "automate_customization",
```

Comment:
So, I think that a) I like this
b) did you want to start getting the code in place to do the navigation depending on the type of the destination? My idea is that you can do something like
```python
if isinstance(destination, basestring):
    force_navigate(destination):
else:
    navigate_to(destination[0], destination[1])
```
or something like that.....